upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1411,6 +1411,7 @@ func (n *nodeUpConfigBuilder) BuildConfig(ig *kops.InstanceGroup, apiserverAddit
 		}
 
 		if isMaster || usesLegacyGossip {
+			config.Channels = n.channels
```

Comment:
I think this is needed only on master nodes.

Reply:
I didn't see logic to keep it from being passed on gossip worker nodes